lib/gems/pending/util/miq-password.rb

```diff
@@ -88,6 +89,10 @@ def self.sanitize_string!(s)
     s.gsub!(REGEXP, '********')
   end
 
+  def self.sanitize_encoded_string(s)
```

Comment:
Any reason to add a new method rather than add this support to the two existing methods above?